### DISCUSSION TITLE
Possible exploit?

### DISCUSSION THREAD

**Discussion by KoltiW0lti** [3kh0/ext-remover#864] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
So I installed the .deb file for Android Studio on my school Chromebook and it showed that I could install it with Linux. But as usual it says "failed to retrieve app info." However I saw on the left side it showed a Linux files folder and it was empty. I also had an SD card installed for whatever reason and i right clicked it and it showed that I could share it with linux. SLinuxid and I right clright-clickedn and it showed that i could manage the usb devices and I got into the linux settings. Is there anyway this could be exploted?

**Comment by akabutnicer** [comment, marked as answer]
No, this is usual. I've had this happen before like a year ago, this is simply intended behavior as Chrome prevents the file from being downloaded with Linux, due to crostini not actually being installed.

**Comment by akabutnicer** [comment]
@KoltiW0lti Also, if you shared with Linux, go to `chrome://os-settings/crostini/androidAdb`

**Comment by akabutnicer** [comment]
Might be useful in your case, not so sure though.

**Reply by KoltiW0lti** [reply]
Sadly I can't turn it on, but thanks!👍

**Reply by akabutnicer** [reply]
No, send a picture of that page, I'm sure you can do something there with what you have available.

**Comment by KoltiW0lti** [comment]
Erm.. its been forever chat..









im so silly <3
